src/Export/SalesChannelService.php

```diff
@@ -58,10 +61,18 @@ public function getSalesChannelContext(
         /** @var FinSearchConfigEntity $systemConfigEntity */
         foreach ($systemConfigEntities as $systemConfigEntity) {
             if ($systemConfigEntity->getConfigurationValue() === $shopkey) {
+                $options = [
+                    SalesChannelContextService::LANGUAGE_ID => $systemConfigEntity->getLanguageId(),
+                ];
+                if (method_exists($currentContext, 'getCurrencyId')) {
```

Comment:
Use `$currentContext->getSalesChannel()->getCurrencyId()`. This way you do not need this `method_exists` check.